src/popup.svelte

```diff
@@ -0,0 +1,106 @@
+<script lang="ts">
+  import { onMount } from 'svelte';
+
+  import { Constants } from './constants';
+  import type { Config } from './constants';
+  import { PlantUmlEncoder } from './encoder/plantUmlEncoder';
+
+  let config: Config = { ...Constants.defaultConfig };
+  let inputUrl = '';
+  let versionPumlSrc = '';
+  let errorMessage = '';
+  let loading = false;
+
+  onMount(() => {
+    chrome.runtime.sendMessage({ command: Constants.commands.getConfig }, (initialConfig: Config) => {
+      config = { ...initialConfig };
```

Comment:
```suggestion
      config = initialConfig;
```

I think referring to `initialConfig` is safe.